### PR TITLE
[13.0][MIG] hr_timesheet_task_required

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

File: hr_timesheet_task_required/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Hr Timesheet Task Required",
+    "summary": """
+        Set task on timesheet as a mandatory field""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV, " "Brainbean Apps, " "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/hr-timesheet",
```
Comment (astirpe) [inline hr_timesheet_task_required/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/timesheet",
```

File: hr_timesheet_task_required/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2018-2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Hr Timesheet Task Required",
+    "summary": """
+        Set task on timesheet as a mandatory field""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV, " "Brainbean Apps, " "Odoo Community Association (OCA)",
```
Comment (astirpe) [inline hr_timesheet_task_required/__manifest__.py]:
```suggestion
    "author": "ACSONE SA/NV, Brainbean Apps, Odoo Community Association (OCA)",
```

**Review by astirpe — APPROVED** [review]
LGTM

**Comment by Saran440** [comment]
@astirpe Fixed. Thank you for your reviewed :)

**Review by alexey-pelykh — APPROVED** [review]

**Pull request merged**